Sources/SwiftyRequest/RestRequest.swift

```diff
@@ -740,3 +756,32 @@ public enum RestError: Error, CustomStringConvertible {
         }
     }
 }
+
+// URL Session extension
+extension RestRequest: URLSessionDelegate {
+    
+    /// URL session function to allow trusting certain URLs
+    public func urlSession(_ session: URLSession, didReceive challenge: URLAuthenticationChallenge, completionHandler: @escaping (URLSession.AuthChallengeDisposition, URLCredential?) -> Void) {
+        let method = challenge.protectionSpace.authenticationMethod
+        let host = challenge.protectionSpace.host
+        switch (method, host) {
+        case (NSURLAuthenticationMethodServerTrust, self.url):
+            #if MAC_OS_X_VERSION_10_6
+                let trust = challenge.protectionSpace.serverTrust
+                if let trust = trust {
+                    let credential = URLCredential(trust: trust)
+                    completionHandler(.useCredential, credential)
+                } else {
+                    print("Attempting to establish a secure connection; no server trust established. Resorting to default handling.")
```

Comment:
@ShihabMehboob Let's replace all print statements with logging statements that use our LoggerAPI. That way we can assign a log level to each log statement. As reference, see what we did for FileKit: https://github.com/IBM-Swift/FileKit/blob/master/Package.swift#L16, https://github.com/IBM-Swift/FileKit/blob/master/Sources/FileKit/FileKit.swift#L44.